### ISSUE TITLE
hook up subscriber

### ISSUE THREAD

**Issue by wingsking** [graphql-dotnet/graphql-dotnet#629]
I know that I can create a subscription like this.

```
public Subscription()
        {
            AddField(new EventStreamFieldType
            {
                Name = "commentAdded",
                Type = typeof(CommentType),
                Subscriber = new EventStreamResolver<Comment>(Subscribe)
            });
        }

private IObservable<Comment> Subscribe(ResolveEventStreamContext context)
        {
            return someIObservableObject;
        }
```

But how can I achieve the same if I want to define a subscription with GraphQL schema language?
I understand that for Query and Mutation, we can do something like below, which hooks up resolver with the GraphQLMetadata attribute. Is there a way to do the same for Subscription?

```
public class Droid
{
  public string Id { get; set; }
  public string Name { get; set; }
}

public class Query
{
  [GraphQLMetadata("hero")]
  public Droid GetHero()
  {
    return new Droid { Id = "123", Name = "R2-D2" };
  }
}

var schema = Schema.For(@"
  type Droid {
    id: String
    name: String
  }

  type Query {
    hero: Droid
  }
", _ => {
    _.Types.Include<Query>();
});

var result = schema.Execute(_ =>
{
  _.Query = "{ hero { id name } }";
});

```

**Comment by wingsking** [comment]
Please take a look at the PR when you have a chance. @pekkah @joemcbride

**Comment by joemcbride** [comment]
Resolved in #647
